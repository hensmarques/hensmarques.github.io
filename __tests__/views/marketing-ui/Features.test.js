import React from 'react';
import Features from '../../../src/views/marketing-ui/Features';

describe('<Features/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Features />);
    expect(wrapper).toMatchSnapshot();
  });
});
