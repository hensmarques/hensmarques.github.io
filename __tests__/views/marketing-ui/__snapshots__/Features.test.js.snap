// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Features/> renders and matches the snapshot 1`] = `
<div
  id="features"
>
  <Fade
    left={true}
  >
    <div
      className="feat-list-content"
    >
      <img
        className="content-icon"
        src="images/icons/SiteIcons2-01.svg"
      />
      <h5
        className="subtitle force-flex-break"
      >
        Next Gen Crypto Exchange
      </h5>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam porttitor, nunc nec mattis finibus, nulla eros suscipit nibh
      </p>
    </div>
  </Fade>
  <Fade
    right={true}
  >
    <div
      className="feat-list-content"
    >
      <img
        className="content-icon"
        src="images/icons/SiteIcons2-02.svg"
      />
      <h5
        className="subtitle force-flex-break"
      >
        Market Data Streaming
      </h5>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam porttitor, nunc nec mattis finibus, nulla eros suscipit nibh
      </p>
    </div>
  </Fade>
  <Fade
    left={true}
  >
    <div
      className="feat-list-content"
    >
      <img
        className="content-icon"
        src="images/icons/SiteIcons2-03.svg"
      />
      <h5
        className="subtitle force-flex-break"
      >
        Advanced Financial Integrations
      </h5>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam porttitor, nunc nec mattis finibus, nulla eros suscipit nibh
      </p>
    </div>
  </Fade>
  <Fade
    right={true}
  >
    <div
      className="feat-list-content"
    >
      <img
        className="content-icon"
        src="images/icons/SiteIcons2-04.svg"
      />
      <h5
        className="subtitle force-flex-break"
      >
        Modular Design
      </h5>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam porttitor, nunc nec mattis finibus, nulla eros suscipit nibh
      </p>
    </div>
  </Fade>
</div>
`;
